### DISCUSSION TITLE
Next.js images domains wildcard syntax

### DISCUSSION THREAD

**Discussion by mhesham32** [vercel/next.js#19942] [Ideas]
for the new next/image component we need to add the hostnames of the images cdn sometimes this becomes annoying because for example on one project I work on we are getting images from the Instagram cdn which could have a different subdomain for newly uploaded images so my settings ended up like this 
```
images: {
    domains: [
      'source.unsplash.com',
      'firebasestorage.googleapis.com',
      'lh3.googleusercontent.com',
      'scontent.cdninstagram.com',
      'video.cdninstagram.com',
      'scontent-hbe1-1.cdninstagram.com',
      'video-hbe1-1.cdninstagram.com',
    ],
  },
```
is there is a way to set Instagram hostnames like this 
```
 '*.cdninstagram.com'
```
thanks

**Comment by FDiskas** [comment]
example PR created https://github.com/vercel/next.js/pull/27345
